shaders/CompoundCloudPlane.shader

```diff
@@ -12,10 +12,10 @@ uniform vec4 colour4 : hint_color  = vec4(0, 0, 0, 0);
 uniform vec2 UVoffset = vec2(0, 0);
 
 // Setting this too high makes the clouds invisible
-const float CLOUD_DISSIPATION = 2.0;
+const float CLOUD_DISSIPATION = 4.0;
 
 // Should be the same as its counterpart in CompoundCloudPlane.cs
-const float CLOUD_MAX_INTENSITY_SHOWN = 1000f;
+const float CLOUD_MAX_INTENSITY_SHOWN = 700f;
```

Comment:
Why change this value? This value is here because Godot clamps the colors to values between 0.0 and 1.0 so we need a way to transform the density data into that constraint to pass it to the shader. It shouldnt affect the end results excepts maybe in terms of numeric precision or something